api/proto/common/common.proto

```diff
@@ -3,8 +3,47 @@ package common;
 option go_package = "github.com/Layr-Labs/eigenda/api/grpc/common";
 
 message G1Commitment {
-	// The X coordinate of the KZG commitment. This is the raw byte representation of the field element.
-	bytes x = 1;
-	// The Y coordinate of the KZG commitment. This is the raw byte representation of the field element.
-	bytes y = 2;
+  // The X coordinate of the KZG commitment. This is the raw byte representation of the field element.
+  bytes x = 1;
+  // The Y coordinate of the KZG commitment. This is the raw byte representation of the field element.
+  bytes y = 2;
 }
+
+/////////////////////////////////////////////////////////////////////////////////////////////////
+// Messages for WIP RPCs. These are subject to change, do not consider these to be stable API. //
+/////////////////////////////////////////////////////////////////////////////////////////////////
+
+// A unique identifier for a blob.
+message BlobKey {
+  oneof key {
+    AuthHeaderBlobKey auth_header_blob_key = 1;
```

Comment:
I'm proposing renaming `BlobAutherHeader` to `BlobHeader` and `BLobHeader` to `BlobCertificate`. If we do this, we could call this field `BlobHeaderBlobKey`. 

https://www.notion.so/eigen-labs/Unified-blob-addressal-in-EigenDA-ae426c610f9e4e61b2e46eec76f7ca4d?pvs=4#bde89f03197d45b89188d541b8345547